src/components/pages/home/card/index.tsx

```diff
@@ -16,6 +16,9 @@ export type CardResource = {
   resources?: CardResource[]
   instructor?: any
   background?: string
+  lightBackground?: string
+  featureCardBackground?: string
+  lightFeatureCardBackground?: string
```

Comment:
I'm not sure how to extend a card type but in this situation ([Looked at this](https://stackoverflow.com/questions/41385059/possible-to-extend-types-in-typescript) but didn't get far

Reply:
So this adds the possible backgrounds to the exported card type that we use